src/autoscaler/db/sqldb/appmetric_sqldb.go

```diff
@@ -26,7 +28,7 @@ func NewAppMetricSQLDB(dbConfig db.DatabaseConfig, logger lager.Logger) (*AppMet
 		return nil, err
 	}
 
-	sqldb, err := sqlx.Open(database.DriverName, database.DSN)
+	sqldb, err := otelsqlx.Open(database.DriverName, database.DSN, otelsql.WithAttributes(database.OTELAttribute))
```

Comment:
Have you tested this somewhere e2e? Just want to check the OTel stuff in Dynatrace but don't know which instance to check :P Feel free to send me the link in a private chat or so to not expose the environments here on GH : D